x/profiles/spec/05-events.md

```diff
@@ -110,32 +110,63 @@ The profiles module emits the following events:
-| message          | sender                         | {userAddress}                         |
+| **Type**                 | **Attribute Key**              | **Attribute Value**                   | 
+|:-------------------------|:-------------------------------|:--------------------------------------|
+| created_application_link | user                           | {userAddress}                         |
+| created_application_link | application_name               | {applicationName}                     | 
+| created_application_link | application_username           | {applicationUsername}                 |
+| created_application_link | application_link_creation_time | {creationTime}                        |
+| message                  | module                         | profiles                              | 
+| message                  | action                         | desmos.profiles.v3.MsgLinkApplication |
+| message                  | sender                         | {userAddress}                         |
 
 ## MsgUnlinkApplication
 
-| **Type**           | **Attribute Key**    | **Attribute Value**                     | 
-|:-------------------|:---------------------|:----------------------------------------|
-| unlink_application | user                 | {userAddress}                           |
-| unlink_application | application_name     | {applicationName}                       | 
-| unlink_application | application_username | {applicationUsername}                   |
-| message            | module               | profiles                                | 
-| message            | action               | desmos.profiles.v3.MsgUnlinkApplication |
-| message            | sender               | {userAddress}                           |
+| **Type**                 | **Attribute Key**    | **Attribute Value**                     | 
+|:-------------------------|:---------------------|:----------------------------------------|
+| deleted_application_link | user                 | {userAddress}                           |
+| deleted_application_link | application_name     | {applicationName}                       | 
+| deleted_application_link | application_username | {applicationUsername}                   |
+| message                  | module               | profiles                                | 
+| message                  | action               | desmos.profiles.v3.MsgUnlinkApplication |
+| message                  | sender               | {userAddress}                           |
 
 ## Keeper
 
+### Chain Link Saved
+
+| **Type**         | **Attribute Key**           | **Attribute Value** | 
+|:-----------------|:----------------------------|:--------------------|
+| saved_chain_link | chain_link_owner            | {userAddress}       |
+| saved_chain_link | chain_link_chain_name       | {chainName}         | 
+| saved_chain_link | chain_link_external_address | {externalAddress}   |
```

Comment:
It is not true since this event is only emitted by msg_server.
https://github.com/desmos-labs/desmos/blob/master/x/profiles/keeper/msg_server_chain_link.go#L32